lib/model/autocomplete.dart

```diff
@@ -799,22 +1033,77 @@ extension WildcardMentionOptionExtension on WildcardMentionOption {
 ///
 /// An instance of this class is managed by [AutocompleteViewManager].
 class AutocompleteDataCache {
-  final Map<int, String> _normalizedNamesByUser = {};
+  final Map<int, String> _lowercaseNamesByUser = {};
```

Comment:
autocomplete [nfc]: Renames/doc changes for explicitness about lowercase

I think the reason to call these fields and methods "normalize" rather than "lowercase" is to anticipate having them be normalized for diacritics as well as for case.